### PR TITLE
[FAB-17765] add R/W in marbles02 init() for system test

### PR DESCRIPTION
**Pull request by dongmingh**
This PR is to add READ/WRITE READ/WRITE in init() of marbles02
A example is given in PTE/marblesInputs
If args length is not 4, then init() will perform no-op as it does today

Signed-off-by: Dongming Hwang <dongming@ibm.com>

### CODE REVIEW

File: chaincodes/marbles02/go/marbles_chaincode.go [hunk 1]
```diff
@@ -115,7 +115,47 @@ func main() {
...
+        marbleName := args[0]
+        color := strings.ToLower(args[1])
+        owner := strings.ToLower(args[3])
+        size, err := strconv.Atoi(args[2])
+        if err != nil {
+                return shim.Error("3rd argument must be a numeric string")
+        }
+
+        // ==== Check if marble already exists ====
+        queryString := fmt.Sprintf("{\"selector\":{\"docType\":\"marble\",\"owner\":\"%s\",\"size\":\"35\"}}", owner)
```
Comment (lindluni) [inline chaincodes/marbles02/go/marbles_chaincode.go]: You can use backticks to escape the entire string so you dont have to escape the quotes 
```fmt.Sprintf(`{"selector" {"docType":"marble","owner":"%s","size":"35"}}`, owner)```

File: chaincodes/marbles02/go/marbles_chaincode.go [hunk 2]
```diff
@@ -115,7 +115,47 @@ func main() {
...
+
+        queryResults, err := getQueryResultForQueryString(stub, queryString)
+        if err != nil {
+		fmt.Println("queryResults ", queryResults)
+                return shim.Error(err.Error())
+        }
+
+
```
Comment (lindluni) [inline chaincodes/marbles02/go/marbles_chaincode.go]: Remove extra blank line

**Review by lindluni — CHANGES_REQUESTED** [review]
Couple real quick comments, can you run `gofmt -l -s -w chaincodes/marbles02/go/marbles_chaincode.go` to properly format the file.

Thanks!

**Comment by dongmingh** [comment]
> Couple real quick comments, can you run `gofmt -l -s -w chaincodes/marbles02/go/marbles_chaincode.go` to properly format the file.
> 
> Thanks!

done, thx

**Review by lindluni — APPROVED** [review]

**Pull request merged**